js/ccex.js

```diff
@@ -131,6 +133,48 @@ module.exports = class ccex extends Exchange {
         return this.parseOrderBook (orderbook, undefined, 'buy', 'sell', 'Rate', 'Quantity');
     }
 
+    async fetchOrderBooks(symbols = undefined, params = {}){
+        await this.loadMarkets ();
+        let orderBooks = [];
+        let orderBooksResult = [];
+
+        try {
+            let response = await this.publicGetFullorderbook();
+            if(response && response.success && response.result) {
+                for (let type of Object.keys(response.result)) {
+                    for (let order of response.result[type]) {
+                        let index = orderBooks.findIndex(function (f){
+                            return f.symbol === order.Market.replace("-", "/").toUpperCase()
+                        });
+
+                        if (index < 0) {
+                            orderBooks.push({
+                                symbol: order.Market.replace("-", "/").toUpperCase(),
+                                bids: [],
+                                asks: []
+                            });
+                            index = orderBooks.length - 1;
+                        }
+
+                        if (type === "buy") {
+                            orderBooks[index].bids.push([order.Rate, order.Quantity]);
+                        } else if (type === "sell") {
+                            orderBooks[index].asks.push([order.Rate, order.Quantity]);
+                        }
+                    }
+
+                    for(let orderBook of orderBooks) {
```

Comment:
`for .. of ..`  constructs are not supported yet, we have to unfold them to basic counter-loops...